Rubberduck.CodeAnalysis/Inspections/Concrete/SuspiciousLetAssignmentInspection.cs

```diff
@@ -0,0 +1,180 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Rubberduck.Inspections.Abstract;
+using Rubberduck.Inspections.Inspections.Extensions;
+using Rubberduck.Inspections.Results;
+using Rubberduck.Parsing;
+using Rubberduck.Parsing.Grammar;
+using Rubberduck.Parsing.Inspections;
+using Rubberduck.Parsing.Inspections.Abstract;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.Parsing.VBA;
+using Rubberduck.Parsing.VBA.DeclarationCaching;
+using Rubberduck.Resources.Inspections;
+using Rubberduck.VBEditor;
+
+namespace Rubberduck.Inspections.Concrete
+{
+    /// <summary>
+    /// Identifies assignments without Set for which both sides are objects.
+    /// </summary>
+    /// <why>
+    /// Whenever both sides of an assignment without Set are objects, there is an assignment from the default member of the RHS to the one on the LHS.
+    /// Although this might be intentional, in many situations it will just mask an erroneously forgotten Set. 
+    /// </why>
+    /// <example hasResult="true">
```

Comment:
the analyzer will flag the example element as having no `hasresult` attribute now.

Reply:
We really expect all lower case? That is rather surprising.

Reply:
Yeah... I thought the analyzer could be used to enforce consistency; I guess it's the casing of the xml elements that compelled me to go with lowercase.. I'll change that to a warning.

Reply:
Just a thought. You might want to consider using some nuget package that does POCO<->XML serialization so that you can just name it `hasResult` in the POCO and it'll be just that. That would also make it easier to extend the xmldocs.